Assets/MRTK/Core/Providers/InputSimulation/SimulatedMotionController.cs

```diff
@@ -0,0 +1,173 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Snapshot of simulated motion controller data.
+    /// </summary>
+    public class SimulatedMotionControllerData
+    {
+        [SerializeField]
+        private bool isTracked = false;
+        public bool IsTracked => isTracked;
+        [SerializeField]
+        private bool isSelecting = false;
+        public bool IsSelecting => isSelecting;
+        [SerializeField]
+        private bool isGrabbing = false;
+        public bool IsGrabbing => isGrabbing;
+        [SerializeField]
+        private bool isPressingMenu = false;
+        public bool IsPressingMenu => isPressingMenu;
+
+        public Vector3 Position { get; set; } = Vector3.zero;
+        public Quaternion Rotation { get; set; } = Quaternion.identity;
```

Comment:
It could be helpful to have some rotation to the controller initially to mirror the orientation of the controllers when they are held.  Maybe something like the gifs below?  It would be consistent with the initial orientation of the simulated hand.




| Current| Slight Rotation|
|---|---|
| ![MotionController](https://user-images.githubusercontent.com/53493796/90555342-12f48400-e14c-11ea-8ab5-11a8e89f91e6.gif) | ![MotionControllerRotate](https://user-images.githubusercontent.com/53493796/90555362-17b93800-e14c-11ea-9157-bd173dd4854e.gif) |

Reply:
+1 on this -having consistency with the initial rotation matching the hands seems good.

Reply:
Change made!